test/kubemark/start-kubemark.sh

```diff
@@ -20,18 +20,19 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-TMP_ROOT="$(dirname "${BASH_SOURCE[@]}")/../.."
-KUBE_ROOT=$(readlink -e "${TMP_ROOT}" 2> /dev/null || perl -MCwd -e 'print Cwd::abs_path shift' "${TMP_ROOT}")
+# @RobertKielty : temporarily commenting this out to see if the local way of getting kube root works here.
```

Comment:
I have no context on WTF is happening here :)

There has to be a simpler way

Reply:
Agreed, I was interested in exploring why this method was chosen in case I was missing something.